### PR TITLE
Start sequence, timer, and score

### PR DESCRIPTION
**Pull request by Bran-Leger**
Implements a basic starting sequence, allowing the user to change the game options. Once the button is clicked, the 3 2 1 countdown begins and then the game starts. Timer is 60 seconds long and the score goes up by 10.

### CODE REVIEW

File: play/speedy-math2/js/app.js [hunk 1]
```diff
@@ -209,12 +229,17 @@ function newProblem() {
     let [ operandOne, operandTwo ] = getOperands(operation);
 
     // set the problem display to the newly generated problem
-    problemEl.innerHTML = `${operandOne} ${operation.sign} ${operandTwo} = ?`;
+    if (operation.type != "exponent") {
+      problemEl.innerHTML = `${operandOne} ${operation.sign} ${operandTwo} = ?`;
+    }
+    else {
+      // TODO Display exponent problem with special formatting
```
Comment (tecanal) [inline play/speedy-math2/js/app.js:236]: You'll most likely have to use the ```<sup>``` tag, which is nice for formatting, but it'll be annoying to strip out when you are trying to calculate the answer.

File: play/speedy-math2/js/app.js [hunk 2]
```diff
@@ -317,9 +342,67 @@ function getRandomNumber(min, max, type) {
...
+
+          timeRemaining--;
+          timerEl.innerHTML = "Time: " + timeRemaining;
+    }, 1000);
+}
+
+/**
+ * Toggles the problem, timer, and score elements
+ */
+function toggleProblems(){
```
Comment (tecanal) [inline play/speedy-math2/js/app.js:403]: Can you also make it focus on the input element so the user doesn't have to click on the input element when they enter their first answer?

**Review by tecanal — CHANGES_REQUESTED** [review]
Once you make the input element autofocus after the countdown, it'll be perfect

File: play/speedy-math2/js/app.js (hunk 1 above)
Comment (Bran-Leger) [reply play/speedy-math2/js/app.js:236]: Thanks for the heads up

File: play/speedy-math2/js/app.js (hunk 2 above)
Comment (Bran-Leger) [reply play/speedy-math2/js/app.js:403]: Changes made

**Pull request merged**